docs/main/concepts/data-engines/local-storage.md

```diff
@@ -85,17 +85,16 @@ A quick summary of the steps to restore include:
    velero restore create rbb-01 --from-backup bbb-01 -l app=test-velero-backup
    ```
 
-## Limitations (or Roadmap items) of OpenEBS Local Storage
+## Limitations of OpenEBS Local Storage
 
 - Size of the Local Storage cannot be increased dynamically.
```

Comment:
- OpenEBS Local Storage is not highly available and cannot sustain node failure.
- OpenEBS Local PV Hostpath does not support snapshots and clones.